testing/sdk_utils.py

```diff
@@ -120,6 +121,18 @@ def is_strict_mode():
     return os.environ.get('SECURITY', '') == 'strict'
 
 
+def is_channel(channel):
+    '''Determine if the tests are being run on a DC/OS cluster with a particular channel.'''
+    active_channel = os.environ.get('DCOS_LAUNCH_CONFIG_CHANNEL', '')
+    return channel and active_channel and re.match(channel, active_channel)
```

Comment:
Is `re.match` not overkill here? Don't we just want `channel == active_channel`?

Reply:
`DCOS_LAUNCH_CONFIG_CHANNEL` gets appended the cluster security mode, e.g. `testing/master/strict`.